mino/minogrpc/server_test.go

```diff
@@ -104,11 +103,10 @@ loop:
 	}
 
 	server.grpcSrv.GracefulStop()
-	err = server.httpSrv.Shutdown(context.Background())
 	require.NoError(t, err)
 }
 
-func Test_ErrorsSimpleCall(t *testing.T) {
+func TestErrorsSimpleCall(t *testing.T) {
```

Comment:
I try to name my tests using this format: TestTypeName_FunctionName. It is easier to find to which piece of code it is related. Here it could be: TestRPC_Call. What do you think, could we add that to the guidelines ?

Reply:
yep, makes sense!